shell/platform/windows/keyboard_key_embedder_handler_unittests.cc

```diff
@@ -942,7 +942,7 @@ TEST(KeyboardKeyEmbedderHandlerTest,
       VK_NUMLOCK, kScanCodeNumLock, WM_KEYDOWN, 0, true, false,
       [&last_handled](bool handled) { last_handled = handled; });
   EXPECT_EQ(last_handled, false);
-  EXPECT_EQ(results.size(), 3);
+  EXPECT_EQ(results.size(), 4);
```

Comment:
Increase the expected value by 1 to account for the post synchronize event.

Reply:
Can you list the two extra events out as expect? I think it's getting confusing.